cps/CP141/CP141.md

```diff
@@ -0,0 +1,101 @@
+
+---
+cp: 141
+title: Initiate the development of Centrifuge V3, a multi-chain, EVM based protocol
+authors: @jeroen

```

Comment:
The "@" must be removed from in front of the usernames, otherwise the header is not rendered properly.